### DISCUSSION TITLE
How to serve script from node_modules to a page using next?

### DISCUSSION THREAD

**Discussion by Rulikkk** [vercel/vercel#4406] [Help] [answered]
I have a script sitting inside node_modules.

I need it to be in the head, e.g. `<script src="node_modules/..."/>` for non-index page; I know how to use "next/head". 

But Next doesn't serve from "node_modules" folder. Requiring that script gives "window not defined" error—it's browser-only script.

Please help. Latest NextJS version used.

**Comment by paulogdm** [comment, marked as answer]
Hello,

This is not supported in Vercel since the `node_modules` is not available after the app is built. For further Next.js assistance, refer to https://github.com/zeit/next.js/discussions
